pkg/urbit/vere/io/term.c

```diff
@@ -485,11 +485,39 @@ _term_it_set_line(u3_utty* uty_u,
                   c3_w     sap_w)
 {
   u3_utat* tat_u = &uty_u->tat_u;
-  u3_noun    txt = u3do("tuft", u3i_words(wor_w, lin_w));
-  c3_w     byt_w = u3r_met(3, txt);
   c3_y*    hun_y = (c3_y*)lin_w;
+  c3_w     byt_w = 0;
 
-  u3r_bytes(0, byt_w, hun_y, txt);
+  //  convert lin_w in-place from utf-32 to utf-8
+  //
+  //    (this is just a hand-translation of +tuft)
```

Comment:
Nice! Does this feel faster?

Any reason not to throw this into a jet?

Reply:
It's hard to tell from just mashing the keyboard locally. I need to run this on a slower cloud VM, I expect it'll reduce or remove the prompt flickering that used to occur.

I lot of stuff in Arvo's text output codepath could stand to be jetted. I'd rather not call a jet here even if I had one, since I want to mutate the buffer in place rather than reallocate. But this could be factored into a function used here and in a jet. This conversion could also be performed earlier in the process, while building up the initial char buffer and interpolating escape codes, but I'm not sure that would make much of a difference.

Note also that I didn't rewrite the inverse translation in `+taft` -- that's conditionally called if the (utf-8) input requires it, so it seems like less of a priority. But it wouldn't be too hard.